### ISSUE TITLE
How to run this code?

### ISSUE THREAD

**Issue by fhb123** [funcwj/nn-ideal-mask#2]
I tried to use you model but i got stuck. Can you tell me how to run the code and more detailed instruction ? thank you very much

**Comment by funcwj** [comment]
This repo is out of date since it was implemented one year ago. But training a mask model is simple, following steps as:
1. got training data and prepare features & target masks (irm, psm, ibm, etc).
2. implement nnet.py dataset.py trainer.py in PyTorch and start training your model.
3. compute output of mask models and conduct monaural speech enhancement.

And you must be familiar with a kind of DL tookit, e.g., PyTorch and got some basic knowledge of speech enhancement (spectrogram, mask. etc).

**Comment by fhb123** [comment]
谢谢您的回复，我的研究领域是语音增强，想要在您的代码基础上进行改进，但是复现代码还是有困难，您能给出具体步骤以及所需的环境配置吗

**Comment by funcwj** [comment]
你先尝试着自己参考代码做吧，有什么具体问题可以再讨论，你现在的问题我觉得太宽泛了。

**Comment by Vichoko** [comment]
This very good code lack of documentation. It was impossible for me to run it.
Also a requirements.txt could be great.
